lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/ActiveContextBridge.java

```diff
@@ -62,7 +62,7 @@ public String getEvaluatorId() {
   }
 
   public void submitTaskString(final String taskConfigurationString) {
-    if (taskConfigurationString.isEmpty()) {
+    if (taskConfigurationString == null || taskConfigurationString.isEmpty()) {
```

Comment:
You can use [`StringUtils.isNotEmpty()`](https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isNotEmpty%28java.lang.String%29)